src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -791,7 +791,7 @@ private static void EncodeRid(List<byte> encodedData, ref BigInteger rid)
             Stack<byte> littleEndianBytes = new Stack<byte>();
             byte continuance = 0;
 
-            while (unencoded != BigInteger.Zero)
+            do
```

Comment:
@steveharter Nah, this code is only used by non-Windows.  On Windows we use CryptEncodeObject... though I do want to make some public API based on (a cleaned up version of) it one of these days.  It's sometimes handy.